packages/dd-trace/src/plugins/util/test.js

```diff
@@ -116,6 +117,35 @@ function getPkgManager () {
   }
 }
 
+function validateMetadata (metadata) {
+  if (GIT_REPOSITORY_URL in metadata && metadata[GIT_REPOSITORY_URL]) {
```

Comment:
nitpick: I think this should be enough 
```suggestion
  if (metadata[GIT_REPOSITORY_URL]) {
```